scripts/deploy.sh

```diff
@@ -138,6 +144,11 @@ if [[ ! -f /etc/openstack_deploy/user_osa_secrets.yml ]] && [[ -f /etc/openstack
   mv /etc/openstack_deploy/user_secrets.yml /etc/openstack_deploy/user_osa_secrets.yml
 fi
 
+#Add Magnum password entries if deploying Magnum
+if [[ "$DEPLOY_MAGNUM" == "yes" ]]; then
+  cat $OA_DIR/playbooks/roles/os_magnum/extras/user_secrets_magnum.yml >> /etc/openstack_deploy/user_osa_secrets.yml
```

Comment:
These should probably be added to a file in the rpc-openstack repo rather than copying it in like this. Not sure though @alextricity25 your thoughts?